src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.head_a {
+  height: 60px;
+}
+
+.header img {
+  margin: 10px 0;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_item {
+  display: inline-block;
+  text-transform: uppercase;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14.06px;
+  transition: color 0.3s;
+}
+
+.nav_item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Vertical margins can be collapsed in some cases. In your case, you've added a right margin to all `.nav_item` elements except the last one, which is a horizontal margin. This is fine, but be careful with vertical margins.